operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -122,11 +191,20 @@ type reconcilePVCExtensionRun struct {
 	client           ctrlClient.Client
 	centralObj       *platform.Central
 	persistence      *platform.Persistence
+	defaultClaimSize resource.Quantity
 	defaultClaimName string
 	target           PVCTarget
 	log              logr.Logger
 }
 
+type PVCOption func(*reconcilePVCExtensionRun)
+
+func WithDefaultClaimSize(value resource.Quantity) PVCOption {
```

Comment:
I think you forgot to use this one.

Reply:
It's sort of intentional. There is no case for changing the default value of the backup volume PVC, and the whole manipulation was mostly to follow the same approach as with the default claim. In fact I'm struggling to come up with a case why would anyone like to change the default claim name either, but still.

Are you suggesting to remove this function?

Reply:
I must be super confused. Can you please explain to me how this works? Here is what I'm seeing:
- `DefaultBackupPVCSize` is unused
- `WithDefaultClaimSize` is not called anywhere
- the `Size` field has no static default
- the `Mul(2)` [here](https://github.com/stackrox/stackrox/pull/14704/files#diff-b1360e9ed38189feb17f7419914315ea2f59b932ec5a046dc3a8d398c32e4450R98) is only called if the user specifies a size explicitly

AFAICT this means that by default the backup PVC size will be the same as the main PVC size, but you indicated that it should be double that size. What am I missng?

Reply:
Damn, it got lost after introduction of `WithOption`. The idea was that the default claim size set before that should depend on the target, which I've done now. As a side note, this doesn't seem to be possible to test such cases now -- the reconcilePVCExtensionRun is prepared by the test itself. Maybe worth pondering how to do that in the future.

Reply:
They cover different aspects:

* getDefaultPVCSizeByTarget is used to set the default (this is probably partly why it's so confusing, it's a "default default value") if no actual value is provided.
* WithDefaultClaimSize allows to specify the value different from the default one. There is no use case for that, it was introduced purely to follow the same standard as with the claim name. I presume you're inclined to remove it, so let me get rid of it.

Reply:
> I expected that the default default would just be a static assignment of DefaultPVCSize - where the ext := reconcilePVCExtensionRun{...} struct is being created.

This is more or less what's happening here. But `DefaultPVCSize` is the default only for the main data volume, for the backup volume we would like to use `DefaultBackupPVCSize`, that's why `getDefaultPVCSizeByTarget` is needed. Does it make sense?

Reply:
I claim this is achievable **without** `getDefaultPVCSizeByTarget`, the last commit in [this PR](https://github.com/stackrox/stackrox/pull/15170) is what I mean.

Reply:
Well, the semantic of what you suggest there is slightly different from what I had in mind -- you assume there is the same default size for both main data volume and the backup volume, where I was planning to have two different defaults. But to be completely honest, I don't care much about how exactly this part would be implemented. Meaning that if you prefer this implementation, I can add it into the PR :)